index.html

```diff
@@ -43,10 +44,12 @@ <h1>Monotone <span>Shapes</span></h1>
     </div>
     <br>
     <div class="config-container">
-        <label for="config">Enter a shape-id (between 1 and 24) or just any text to generate SVG</label>
+        <label style="font-family:'Roboto';" for="config">Enter a shape-id (between 1 and 24) or just any text to generate SVG</label>
```

Comment:
let's not use multiple fonts roboto and Poppins, also its not recommeded inline css